IFC4x3/Properties/n/NumberOfDrillholes_2$SNxqBr14fPN2tHxaHFdx/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfDrillholes_2SNxqBr14fPN2tHxaHFdx" Name="NumberOfDrillholes" UniqueId="bf717ef4-2f50-44a5-95c2-dd1ee444f9fb" PropertyType="p_listvalue" PrimaryDataType="IfcLabel" />
```

Comment:
Propose renaming to "DrillholeCount". Confusing property type (list?) and data type (`IfcLabel`?).